besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/RocksDbUsageHelper.java

```diff
@@ -30,6 +35,37 @@
 public class RocksDbUsageHelper {
   private static final Logger LOG = LoggerFactory.getLogger(RocksDbUsageHelper.class);
 
+  static void forEachColumnFamily(
```

Comment:
This function doesn't really belong in the RocksDbUsageHelper now that it's more generic. Perhaps move it out into a RocksDbHelper class?

Reply:
That's true. Have renamed class now it's not just about Usage